CHANGELOG.md

```diff
@@ -7,134 +7,202 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- Support for `allowedOutdatedData` in `addItems` and `updateItems` functions.
+
 ## [0.11.0] - 2020-12-23
+
```

Comment:
@lucasecdb @wdsrocha our eslint rules changed again? What is the expected behavior, with or without empty lines?

Reply:
eslint does not look to markdown files by default. But if I'm not mistaken, if VSCode is configured to run auto-formatting, inserting spaces between lines is the default formatting.